config/routes.rb

```diff
@@ -156,6 +156,8 @@
         resources :folders, except: [:new, :create, :index] do
           get 'new_subfolder', on: :member, path: 'new/subfolder'
           post 'create_subfolder', on: :member, path: 'create/subfolder'
+          get 'new_materials', on: :member, path: 'new/files'
```

Comment:
@lowjoel I changed to `new/files`, is that better ? I want to present just files to user, and internally I still use material because they are different things in the backend.